packages/herdtools7/herdtools7.7.56.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "herdtools7"
+version: "7.56.1"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+description: ""
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+build: ["sh" "./dune-build.sh" "%{prefix}%"]
```

Comment:
Dune has a subst feature to have automatic development and release versioning of the packages. It would greatly simplify the build and installation

Reply:
Comsider also not using `wrapped false` In the huild, it is very name conflict prone

Reply:
The installation seems fine. No library is installed, just executables, so there is no need to address the build and install points if this is what is expected

Reply:
Ok, thanks, won't fix.